### PR TITLE
BP-697 line length check

### PR DESCRIPTION
**Pull request by saranchuk-hys**

### CODE REVIEW

File: Model/GuestPaymentInformationManagement.php [hunk 1]
```diff
@@ -24,9 +24,8 @@
 use Buckaroo\Magento2\Api\GuestPaymentInformationManagementInterface;
 use Buckaroo\Magento2\Model\ConfigProvider\Method\Factory;
 
-// @codingStandardsIgnoreStart
+// phpcs:ignore
 class GuestPaymentInformationManagement extends MagentoGuestPaymentInformationManagement implements GuestPaymentInformationManagementInterface
-// @codingStandardsIgnoreEnd
```
Comment (florinel-chis) [inline Model/GuestPaymentInformationManagement.php:29]: I would expect a fix, not ignore again

File: Test/Unit/Gateway/Http/TransactionBuilder/AbstractTransactionBuilderMock.php [hunk 2]
```diff
@@ -26,6 +26,7 @@
  *
  * @package Buckaroo\Magento2\Test\Unit\Gateway\Http\TransactionBuilder
  */
+// phpcs:ignore
 class AbstractTransactionBuilderMock extends \Buckaroo\Magento2\Gateway\Http\TransactionBuilder\AbstractTransactionBuilder
```
Comment (florinel-chis) [inline Test/Unit/Gateway/Http/TransactionBuilder/AbstractTransactionBuilderMock.php:30]: this one too... (and some others below)

**Comment by florinel-chis** [comment]
@saranchuk-hys 
in this PR there shouldn't be any phpcs ignore left...

**Comment by Buckaroo-Rens** [comment]
Closed.

**Pull request closed without merging**